changelog.d/20250513_182417_jasonalt.rst

```diff
@@ -0,0 +1,53 @@
+..
+.. A new scriv changelog fragment
+..
+.. Uncomment the header that is right (remove the leading dots).
+..
+.. Leave the "(:pr:`...`)" text in your change description.
+.. GitHub Actions will automatically replace it when the PR is merged.
+..
+.. Python Support
+.. ~~~~~~~~~~~~~~
+..
+.. - A bullet item for the Python Support category. (:pr:`NUMBER`)
+..
+.. Added
+.. ~~~~~
+..
+.. - A bullet item for the Added category. (:pr:`NUMBER`)
+..
+.. Removed
+.. ~~~~~~~
+..
+.. - A bullet item for the Removed category. (:pr:`NUMBER`)
+..
+Changed
+~~~~~~~
```

Comment:
As we march to v4, a new header has been introduced recently: "Breaking Changes". This helps call out the changes.

```suggestion
Breaking Changes
~~~~~~~~~~~~~~~~
```

This might not need to be addressed in this PR because Max is converting EVERYTHING to use `MISSING` and could potentially remove/update your changelog fragment, but I wanted to call it out.